docs/modules/ROOT/pages/documentation/payara-server/server-configuration/security/multiple-mechanism-in-ear.adoc

```diff
@@ -22,6 +22,8 @@ In order to configure a specific `HttpAuthenticationMechanism` for a Web applica
 </context-param>
 ----
 
+NOTE: Implicit CDI scanning for all CDI modules inside the EAR assembly must be disabled or @Veto annotation can be used on class that implement HTTP authentication in order for `HttpAuthenticationMechanism` to work as expected because if two or more CDI beans implement HTTP authentication they will clash and an exception is raised. CDI beans can be defined through web.xml descriptor. Payara Server will turn only defined once into CDI bean for a specific WAR app.
```

Comment:
I think saying that CDI scanning for all modules should be disabled is misleading. It's more correct to state that there must not be any CDI beans that implement `HttpAuthenticationMechanism` in any of the EAR modules. If there are such beans, they need to be disabled either using `@Veto` or excluded from scanning in the beans.xml file as described in the specification here: https://jakarta.ee/specifications/cdi/3.0/jakarta-cdi-spec-3.0.html#exclude_filters, which also applies to CDI 2.0, which is provided by Payara Server.